lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -78,8 +78,8 @@ def process(cube, ensemble_realization_numbers=None):
 
         Args:
             cube (iris.cube.Cube):
-            Cube containing a percentile coordinate, which will be rebadged as
-            ensemble realization.
+                Cube containing a percentile coordinate, which will be
+                rebadged as ensemble realization.
```

Comment:
The keyword arg ensemble_realization_numbers is still not defined.

Reply:
Fixed